### PR TITLE
Main page banner and navbars

### PR DESCRIPTION
**Pull request by GuilhermeJSilva**
- Added main page banner 
- Added customization options to navbar

[Preview](https://feature-landing--friendly-shockley-474156.netlify.com/)

### CODE REVIEW

File: src/styles/banner.module.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  margin-top: 5em;
+}
+
+.icon_links {
+  display: flex;
+  justify-content: center;
+}
+
+.icon_links a {
+  margin: 0 1em;
```
Comment (smbea) [inline src/styles/banner.module.css]:
```suggestion
  margin: 0 0.3em;
```

File: src/components/mainpage/basic-info.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  {
+    icon: <FaEnvelope />,
+    href: "mailto:geral@sinf.pt",
+  },
+]
+
+const BasicInfo = () => {
+  return (
+    <div className={BannerStyles.info}>
+      <p>2º EDIÇÂO</p>
```
Comment (pedromsfernandes) [inline src/components/mainpage/basic-info.js]: Typo here - should be `<p>2ª EDIÇÃO</p`

File: src/components/navbar.js [hunk 3]
```diff
@@ -39,6 +39,9 @@ export default class NavBar extends React.Component {
     super(props)
 
     this.toggle = this.toggle.bind(this)
+    this.fixed = props.fixed
```
Comment (pedromsfernandes) [inline src/components/navbar.js]: Since these are props and we only need them in the render method, do they need to be set as class attributes?

Comment (GuilhermeJSilva) [reply src/components/navbar.js]: How are they to be accessed in the render method otherwise?

**Pull request merged**